umbraco-cloud/set-up/smtp-settings.md

```diff
@@ -28,30 +40,26 @@ Configure _SenderEmail_ in the `appsettings.json` file under `Umbraco:CMS:Global
 },
 ```
 
-{% hint style="info" %}
-**Legacy Umbraco**
-
-If your Cloud project is running Umbraco 7 or 8, the _SenderEmail_ is configured in the `<notifications>` section of the `web.config` file. Find more details on this in the [Legacy Documentation](https://our.umbraco.com/documentation/Reference/Configuration-for-Umbraco-7-and-8/umbracoSettings/).
-{% endhint %}
-
 ### Backoffice Users
 
-There are two scenarios for Backoffice users where configuring an SMTP service is needed:
+Configuring an SMTP service is needed in the following scenarios for Backoffice users:
 
-1. When you want to add a user to your project directly from the Backoffice. Doing this involves sending out an email to the new user. For this scenario, we've set up a fallback, which means that even though you haven't yet configured an SMTP service, an email will still be sent to the new user. Keep in mind that the fallback is **only** for this particular purpose; inviting users to join your project.
-2. To set up the SMTP service for your Umbraco Cloud project if one of your Backoffice users has forgotten their password. To reset their password, they have to request a password reset which will be sent to them by mail. This will only work once you've configured an SMTP service.
+1. When you want to add a user to your project from the Backoffice. This action involves sending an email to the new user. We have set up a fallback that ensures an email is sent even if you haven't configured the SMTP service yet. However, this fallback is **only** for inviting users to your project.
+2. To set up the SMTP service for your Umbraco Cloud project if one of your Backoffice users forgets their password. The user will need to request a password reset, which will be sent via email. This will only work once you've configured the SMTP service.
```

Comment:
This to me looks a little wierd in an ordered list 🤔 
Not sure if an unordered list is better, or if the two scenarios could be set up in a better way 🤔 

Thoughts?

Reply:
Makes sense. I have changed it to an unordered list and refined the text a bit more, so that there's not too much text to read 💪